packages/web/src/components/inbox-unavailable-modal/InboxUnavailableModal.tsx

```diff
@@ -83,55 +92,50 @@ const actionToContent = (
 const { beginTip } = tippingActions
 const { getCanCreateChat } = chatSelectors
 
-export const InboxUnavailableModal = ({
-  isVisible,
-  onDismiss,
-  onClose,
-  user
-}: {
-  isVisible: boolean
-  onDismiss?: () => void
-  onClose: () => void
-  user: User
-}) => {
+export const InboxUnavailableModal = () => {
+  const { isOpen, onClose, onClosed, data } = useInboxUnavailableModal()
+  const { user, presetMessage, onCancelAction } = data
   const dispatch = useDispatch()
   const { callToAction } = useSelector((state) =>
-    getCanCreateChat(state, { userId: user.user_id })
+    getCanCreateChat(state, { userId: user?.user_id })
   )
   const hasAction =
     callToAction === ChatPermissionAction.TIP ||
     callToAction === ChatPermissionAction.UNBLOCK
 
   const handleClick = useCallback(() => {
+    if (!user) return
```

Comment:
can this ever happen? maybe worth a log

Reply:
techincally user is undefined when the modal is closed, but should ever get this far no - adding a console error